opentelemetry-sdk/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Rename Observer to ValueObserver
+  ([#751](https://github.com/open-telemetry/opentelemetry-python/pull/751))
```

Comment:
I would like to call out the update to `Record`'s method signature as well

Reply:
`Record` s are used internally, so I don't believe they need to be exposed to the user and shown in the CHANGELOG.

Reply:
Got it, thanks